requirements-dev.txt

```diff
@@ -7,3 +7,5 @@ pytest-cov
 sphinx
 sphinx_lfs_content
 sphinx_rtd_theme
+ipyleaflet
+ipywidgets
```

Comment:
`requirements-dev.txt` should only contain those dependencies that are *not* needed for the project itself, but for testing, documentation etc.